### DISCUSSION TITLE
intermediate version of osmand

### DISCUSSION THREAD

**Discussion by arncor** [osmandapp/OsmAnd#15630] [General]
How do you decide if you're gonna release a 4.2.x before the next major 4.3 update ?
I see there are milestones for major update 4.x for example, but nothing for 4.2.x

do you have a date for the next 4.3?
would it be possible to release a quick 4.2.8 (for example) with at least the fix for OpenGL crash on launch when starting with a mapstyle that depends on another standard mapstyle ?

thanks :)

**Comment by vshcherb** [comment]
4.3 is almost done, next week will be in beta. Quick releases distract us quite a lot we give 2-3 weeks release period when we release updates then we take a break to prepare next release

**Comment by arncor** [comment]
Ok, thank you for your answer :)
